app/views/homes/index.html.erb

```diff
@@ -1,4 +1,11 @@
-<p>
```

Comment:
You should update the path on this file to be /airports/index.html.erb

Homes is the default that it comes with